clients/drcachesim/tracer/instru_offline.cpp

```diff
@@ -681,7 +681,7 @@ int
 offline_instru_t::instrument_memref(void *drcontext, void *bb_field, instrlist_t *ilist,
                                     instr_t *where, reg_id_t reg_ptr, int adjust,
                                     instr_t *app, opnd_t ref, int ref_index, bool write,
-                                    dr_pred_type_t pred)
+                                    dr_pred_type_t pred, bool memref_needs_full_info_)
```

Comment:
style violation: trailing underscore is only used for member vars.

Ditto in 3 places below.

Reply:
Fixed it. I had forgotten to fix this when I made this variable a param.